pages/Source_Code_Analysis_Tools.md

```diff
@@ -4,47 +4,63 @@ layout: col-sidebar
 title: Source Code Analysis Tools
 author: 
 contributors: 
-  - Dave Wichers, itamarlavender, will-obrien, Eitan Worcel, Prabhu Subramanian, kingthorin, coadaflorin, hblankenship, GovorovViva64, pfhorman, GouveaHeitor, Clint Gibler, DSotnikov, Ajin Abraham, Noam Rathaus   
+  - Dave Wichers, itamarlavender, will-obrien, Eitan Worcel, Prabhu Subramanian, kingthorin, coadaflorin, hblankenship, GovorovViva64, pfhorman, GouveaHeitor, Clint Gibler, DSotnikov, Ajin Abraham, Noam Rathaus, Mike Jang
 tags: source code analysis, static code analysis, tools
 permalink: /Source_Code_Analysis_Tools
 
 ---
 
 {% include writers.html %}
 
-[Source code analysis](Static_Code_Analysis) tools, also referred to as Static Application Security Testing (SAST) Tools, are designed to analyze source code or compiled versions of code to help find security flaws.
+[Source code analysis](Static_Code_Analysis) tools, also known as Static Application Security Testing (SAST) Tools, can help you analyze source code or compiled versions of code to help find security flaws.
 
-Some tools are starting to move into the IDE. For the types of problems that can be detected during the software development phase itself, this is a powerful phase within the development life cycle to employ such tools, as it provides immediate feedback to the developer on issues they might be introducing into the code during code development itself. This immediate feedback is very useful, especially when compared to finding
-vulnerabilities much later in the development cycle.
+You can add some SAST tools into your IDE. Such tools can help you detect issues during software development.
+SAST tool feedback can save time and effort, especially when compared to finding vulnerabilities later in the development cycle.
 
 ## Strengths and Weaknesses
 
 ### Strengths
 
 - Scales well -- can be run on lots of software, and can be run repeatedly (as with nightly builds or continuous integration).
-- Useful for things that such tools can automatically find with high confidence, such as buffer overflows, SQL Injection Flaws, and so forth.
-- Output is good for developers -- highlights the precise source files, line numbers, and even subsections of lines that are affected.
+- Identifies certain well-known vulnerabilities, such as:
```

Comment:
I've limited the list to what's known. If the audience is developers, I think "code snippet" is more appropriate than "subsections of lines"